upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -738,6 +738,14 @@ func (c *ApplyClusterCmd) Run(ctx context.Context) error {
 			return err
 		}
 
+		if cluster.Spec.GetCloudProvider() == kops.CloudProviderScaleway {
+			scwCloud := cloud.(scaleway.ScwCloud)
+			err := tf.AddOutputVariable("zone", terraformWriter.LiteralFromStringValue(scwCloud.Zone()))
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
See how `project` is set for GCE and use the same pattern.
```suggestion
		if scwZone != "" {
			if err := tf.AddOutputVariable("zone", terraformWriter.LiteralFromStringValue(scwZone)); err != nil {
				return err
			}
		}
```

Reply:
Done :+1: